commons/connector/fitbit/fitbit_intraday_heart_rate.avsc

```diff
@@ -7,6 +7,7 @@
     { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
     { "name": "timeReceived", "type": "double", "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)." },
     { "name": "timeInterval", "type": "int", "doc":  "Chronological window size (s)." },
-    { "name": "heartRate", "type": "int", "doc": "Heart rate value (bpm)."}
+    { "name": "heartRate", "type": "int", "doc": "Heart rate value (bpm)."},
+    { "name": "restingHeartRate", "type": "int", "doc": "Resting heart rate value for the day. A sleep stage log is required to generate this value. When a classic sleep log is recorded, this value will be missing." }
```

Comment:
Isn't this generated in the summary instead of part of the intraday heart rate? In any case, whenever a new field is added, please make it nullable as follows: `"type": ["null", "<actual type>"], "default": null`. This reduces incompatibilities with existing data.

Reply:
I notice that the restingHeartRate field is present in both the intraday response and the summary response. Correct me if am wrong. Should I also add this field to `fitbit_activity_heart_rate.avsc`?
- https://dev.fitbit.com/build/reference/web-api/heartrate-timeseries/get-heartrate-timeseries-by-date/
- https://dev.fitbit.com/build/reference/web-api/intraday/get-heartrate-intraday-by-date/

Reply:
Yes, it makes sense to add it to the activity heart rate as well.